src/css/lib/monster.ui.scss

```diff
@@ -944,3 +944,48 @@ i.monster-password-strength {
 	/* position: relative !important; */
 	z-index: 4 !important;
 }
+
+/************************************************************/
+/********************* KEY-VALUE EDITOR *********************/
+/************************************************************/
+.monster-key-value-editor {
+	.monster-key-value-data-container {
+		.monster-key-value-editor-row {
+			margin-bottom: 10px;
+			display: flex;
+
+			&:hover {
+				.key-value-remove {
+					opacity: 1;
+				}
+			}
+
+			.data-key, .data-value {
+				flex: 1;
+				padding-right: 1rem;
+
+				input {
+					width: 100%;
+					box-sizing: border-box;
+					height: 30px;
+				}
+			}
+
+			.key-value-remove {
+				color: $font-primary-color;
+				cursor: pointer;
+				padding-top: 0.2rem;
+				opacity: 0;
+				transition: visibility 0.25s, opacity 0.25s ease-in-out;
```

Comment:
Why transition the visibility as well as the opacity?

I suggest you pick only one property: for this use case, `visibility` seems like the appropriate choice (don't forget to update the property on hover).

Also, the transition speed seems a bit slow, 1/8 of a second should keep thinks snappy.

Reply:
You're right, one seems to be enough. I'll leave opacity, because it adds a fade effect, and visibility is only a blink.